.travis.yml

```diff
@@ -41,7 +41,7 @@ matrix:
     script:
       - cargo install cargo-travis --debug || echo "cargo-travis has been already installed"
       - mkdir target # fix for cargo-coveralls
-      - cargo coveralls
+      - cargo coveralls --exclude-pattern "$PWD/target/"
```

Comment:
What's that for?

Reply:
The coverage builds enable instrumentation via compiler flags which are now forwarded to the build of libsodium. Hence the source code of libsodium is included in the coverage metrics which made them drop by more than 50%. This excludes the libsodium code under `target/debug/libsodium-sys-*/out` from the coverage computation as we are testing the bindings assuming the libsodium implementation to be correct.